docs/io/configuration/components/spectrum.rst

```diff
@@ -15,7 +15,38 @@ just an integer. Finally, the method option selects the final spectral synthesis
 * real: construct spectrum from the real packet population alone
```

Comment:
I think we should link the new page on basic spectrum generation here.

Reply:
If by new page you mean [https://tardis-sn.github.io/tardis/physics/spectrum/basic.html](url), then I think under the virtual construction we should link the page on spectrum generation with virtual packets as well ( [https://tardis-sn.github.io/tardis/physics/spectrum/virtualpackets.html](url) ), unless this page is still under development?